hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeEdge.java

```diff
@@ -202,6 +204,11 @@ public <V> Property<V> property(String key, V value) {
         E.checkArgument(this.label.properties().contains(propertyKey.id()),
                         "Invalid property '%s' for edge label '%s'",
                         key, this.label());
+        if (value == null) {
+            this.removeProperty(propertyKey.id());
```

Comment:
if remove null value's property,  may cause tinkerpop ut failed ?

Reply:
follow the tinkerpop actions with [release-1.0.0](https://github.com/apache/incubator-hugegraph/actions/workflows/ci.yml?query=branch%3Arelease-1.0.0)